src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -853,6 +853,7 @@ subroutine mixedlayer_restrat_Bodner(CS, G, GV, US, h, uhtr, vhtr, tv, forces, d
   logical :: line_is_empty, keep_going
   integer, dimension(2) :: EOSdom ! The i-computational domain for the equation of state
   integer :: i, j, k, is, ie, js, je, Isq, Ieq, Jsq, Jeq, nz
+  logical :: lam2_available = .false.
```

Comment:
I don't see anything about this in the [style guide](https://github.com/mom-ocean/MOM6/wiki/Code-style-guide), but if you are going to initialize `lam2_available` I think you want to do it below rather than in the variable declaration to avoid implicitly `save`-ing it. Maybe just set it to `.false.` in the `else` block for the `if (CS%wave_enhanced_ustar) then` you modify below?